test/conftest.py

```diff
@@ -21,3 +25,29 @@ def mock_tokenizer():
 @pytest.fixture()
 def test_files_path():
     return Path(__file__).parent / "test_files"
+
+
```

Comment:
I left this mock here so high up in the test tree because it is used in multiple places (including e2e tests)